### DISCUSSION TITLE
option to give in start and stop time of a custom added movie

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#295] [Ideas] [answered]
last Tuesday evening the elder who was overseer that evening (don't know if that's the correct English word) asked if it was possible to download the recorded stream and to show a presentation because someone cancelled.
So I started sweating.. downloading 700MB or something. Opening the file finding begin and end, cut the clip, save it and added the clipped file to JWMMF. Managed just in time, we have a decent internet line bandwidth and a new fast PC to encode the clip.

I thought you make use of VLC to play media in JWMMF?

VLC has EXTVLCOPT options that can be used with #EXTVLCOPT:start-time=500 and #EXTVLCOPT:stop-time=900 so VLC shows only second 500 to 900 and then quits. Of course, you know this. :-)
IS it possible (and worth the trouble) to have the option to give in a start and stop time of a mediafile where you only need a small part?

![start-stop](https://user-images.githubusercontent.com/39587825/167275022-825facdd-6cb7-4578-a8a4-1c591eb05b4c.png)

or maybe in the media list generator..

**Comment by sircharlo** [comment, marked as answer]
I like this idea alot!
I'll try to cook something up.
